salt/kubernetes-minion/kubelet.jinja

```diff
@@ -26,7 +26,7 @@ KUBELET_ARGS="\
 {% endif -%}
     --node-ip={{ grains['ip4_interfaces']['eth0'][0] }} \
     --config=/etc/kubernetes/manifests \
-{% if salt['pillar.get']('pod_infra_container_image') -%}
```

Comment:
And wasn't your previous solution working? Because I would expect this to work:

`pillar/kube-minion.sls`:
```
{% if grains['lsb_distrib_id'] == "CAASP" -%}
# infra container to use instead of downloading gcr.io/google_containers/pause
pod_infra_container_image: sles12/pause:1.0.0
{% endif -%}
```

`salt/kubernetes-minion/kubelet.jinja`:
```
{% if salt['pillar.get']('pod_infra_container_image', '')|length > 0 -%}
--pod-infra-container-image={{ pillar['pod_infra_container_image'] }} \
{% endif -%}
```

This way we could always overwrite the pause container, with `sles12/pause:1.0.0` as the default image for CaaSP.